bugbug/bug_features.py

```diff
@@ -209,6 +210,46 @@ def __call__(self, bug, **kwargs):
         return None
 
 
+class delta_nightly_request_merge(single_bug_feature):
+    name = "Time delta between landing of the patch in Nightly and uplift request"
+
+    def __call__(self, bug, **kwargs):
+        for history in bug["history"]:
+            for change in history["changes"]:
+                if not (
+                    change["added"].startswith("approval-mozilla")
+                    and change["added"].endswith("?")
+                ):
+                    continue
+
+                uplift_request_datetime = parser.parse(history["when"])
+
+                landing_comments = Bugzilla.get_landing_comments(
+                    bug["comments"], ["nightly"]
+                )
+
+                # This will help us to find the closest landing before the uplift request
+
+                time_delta = None
+                found = False
```

Comment:
Nit: I would refactor the following code to avoid the need to use the stop flag (i.e., `found`). Making a list of the qualified landings and then using `max()` to get the closest landing before the uplift request could simplify that.

Reply:
Refactored code to list search